docs/_news/news11.md

```diff
@@ -0,0 +1,72 @@
+
+**Codewind Website**
+- We are pleased to announce that our blogs now have a [home](https://www.eclipse.org/codewind/blog.html) on our website! 🏡
+
+**IDEs**
+##### VS Code, Eclipse, and IntelliJ
+- [Appsody projects no longer show build status](https://github.com/eclipse/codewind/issues/2052). 
+
+##### VS Code and Eclipse Che
+- The Codewind extension downloads and activates significantly faster. [The extension size is reduced by 98% to 1.6 MB](https://github.com/eclipse/codewind/issues/1060). 
+- You only need to download the `cwctl` and `Appsody` binaries one time. They are fetched from the internet when the [extension is activated for the first time](https://github.com/eclipse/codewind-vscode/pull/506). 
+- The [Codewind extension starts up faster](https://github.com/eclipse/codewind/issues/2280) now that [the extension is bundled through Webpack](https://github.com/eclipse/codewind-vscode/pull/536).
+
+##### VS Code and Eclipse
+- Image registries are managed on the local connection.
+    - [Projects that pull from private image registries can be created locally](https://github.com/eclipse/codewind/issues/1306). 
+    - There is no push registry for the local connection because [images are managed by Docker Desktop and never leave the user's machine](https://github.com/eclipse/codewind/issues/2588).
+- The [Performance Dashboard is always enabled](https://github.com/eclipse/codewind/issues/2299) even if the project does not have metrics available. 
+    - Note that for [projects without metrics, running loads fail](https://github.com/eclipse/codewind/issues/2384).
+
+##### VS Code
+- You can use the profiling extension for [remote connections](https://github.com/eclipse/codewind/pull/2123) and in [Che](https://github.com/eclipse/codewind-vscode/pull/457).
+- [The **Add to Workspace** project command is reworked for multi-root workspaces](https://github.com/eclipse/codewind-vscode/pull/484). It no longer appears on projects that are already in the root workspace folder and gives an overrideable warning if run on a project that is in a subfolder of the workspace.
+- The homepage is updated so that [the **Create** and **Add** project creation steps are greyed out and unclickable until the actions are allowed](https://github.com/eclipse/codewind/issues/2255). On the **Local** tab, local Codewind must be started. On the **Remote** tab, a remote connection must exist.
+- The **Remote Connection** settings page layout is [improved and more responsive](https://github.com/eclipse/codewind-vscode/pull/476).
+- [Corrections were made to webview links and buttons](https://github.com/eclipse/codewind-vscode/pull/473).
+
+##### Eclipse
+- After you install Codewind for Eclipse and restart the Eclipse IDE, the Eclipse welcome page shows the Codewind for Eclipse link. Clicking this link closes the Eclipse welcome page and opens the Codewind Explorer view and shows the Codewind welcome page in the editor view. 
```

Comment:
Add at the end: See the Quick Start section of the Codewind welcome page to get started with Codewind. The Learn section has useful links for learning more about interacting with Codewind through the IDE.